README.md

```diff
@@ -14,3 +14,11 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+# Answers
+
+- First, the asymptotic analysis of an algorithm could be misleading because the analysis could make it look like the algorithm is quick, but each action in the algorithm could take a lot of time if it is built that way.  Second, as we saw from the data in class, two algorithms with the same time complexity could actually have very different run times.  Third, although the asymptotic analysis could make us think an algorithm is fast or slow, it really just shows us how the runtime changes as the input size or some variable changes.
```

Comment:
What are specific reasons (e.g. things in the definition of $O$) that can cause algorithms with the same time complexity to have different runtimes in practice?